specs/phase0/p2p-interface.md

```diff
@@ -587,11 +587,13 @@ The response MUST consist of zero or more `response_chunk`. Each _successful_ `r
 
 Clients MUST keep a record of signed blocks seen since the since the start of the weak subjectivity period and MUST support serving requests of blocks up to their own `head_block_root`.
 
-Clients MUST respond with at least one block, if they have it and it exists in the range. Clients MAY limit the number of blocks in the response.
+Clients MUST respond with at least the first block that exists in the range, if they have it.
 
-The response MUST contain no more than `count` blocks.
+The following blocks MUST be ordered consecutively, with each `parent_root` matching the `hash_tree_root` of the previous block.
```

Comment:
And what happens if `step > 1`, the parent roots cannot be verified? Should we just say that the blocks are part of the same chain and ordered, but may have gaps to skip slots for the `step > 1` case?

Reply:
right, good catch - we could remove `skip` simply - it's a small optimization that kind of only makes sense on homogenous networks where blocks are served at similar speeds - otherwise, a similar approach is to grab an epoch at a time and keep `step=1` always - it has the same worst-case behaviour.

Reply:
`step` is being used in multiple client sync strategies at the moment. Don't think we should remove it at this point

Reply:
We implement it as well - but it's also the one thing that can be removed without any real loss in worst-case behaviour, far as I can tell.